examples/c/publish_subscribe_cross_language/README.md

```diff
@@ -0,0 +1,110 @@
+# Cross-language Publish-Subscribe
+
+Before proceeding, all dependencies need to be installed. You can find
+instructions in the [C Examples Readme](../README.md).
+
+## Running The Example
+
+This example illustrates a robust publisher-subscriber communication pattern
+between a C publisher and a Rust subcriber application. The C publisher
+application sends a message every second, each containing `TransmissionData` and
+the `CustomHeader`. On the receiving end, the Rust subscriber application prints
+the received payload and the user header to the console whenever new data
+arrives.
+
+First you have to build the C examples:
+
+```sh
+cmake -S . -B target/ffi/build -DBUILD_EXAMPLES=ON
+cmake --build target/ffi/build
+```
+
+To observe this dynamic communication in action, open two separate terminals and
+execute the following commands:
+
+### Terminal 1
+
+Run the Rust subscriber application from the
+[Rust cross-language publish-subscribe example](../../rust/publish_subscribe_cross_language):
+
+```sh
+cargo run --example publish_subscribe_cross_language_subscriber
+```
+
+### Terminal 2
+
+Run the C publisher application:
+
+```sh
+./target/ffi/build/examples/c/publish_subscribe_cross_language/example_c_cross_language_publisher
+```
+
+Feel free to also run the C subscriber and the Rust publisher applications
```

Comment:
Maybe feel free to run any other cross language example written in other languages?